packages/apps/fortune/exchange-oracle/server/src/common/interceptors/transform-enum.interceptor.spec.ts

```diff
@@ -161,4 +161,38 @@ describe('TransformEnumInterceptor', () => {
     });
     expect(callHandler.handle).toHaveBeenCalled();
   });
+
+  it('should return bodyOrQuery if instance is not an object', () => {
+    // Test with `null` as the instance
+    let result = interceptor['lowercaseEnumProperties']( 
```

Comment:
Why not `await interceptor.intercept(executionContext, callHandler).toPromise();` as in other tests and comparing that body is transformed properly for objects w/o nested values?

Reply:
Because this test is directly calling the private method `lowercaseEnumProperties`, rather than executing the full interceptor flow.

Reply:
@dnechay